delivery_package_gs1_barcode_report/reports/stock_report_views.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+        <record id="action_report_picking_packages" model="ir.actions.report">
+            <field name="name">SSCC Labels</field>
+            <field name="model">stock.picking</field>
+            <field name="report_type">qweb-pdf</field>
+            <field
+            name="report_name"
```

Comment:
Why is this a new report and did you not add the barcodes to the regular Package label PDF.

Reply:
Not addressing this one for now, the module is now usable.